kbase-extension/static/kbase/css/contigBrowserStyles.css

```diff
@@ -16,10 +16,10 @@
 /**
  * Adjusts the font size and stroke width of the base-position axis.
  */
-.kbcb-widget .kbcb-axis path,
-.kbcb-widget .kbcb-axis line {
+.kbcb-widget .kbcb-axis line,
+.kbcb-widget .kbcb-axis path {
   fill: none;
-  stroke: black;
+  stroke: #000;
```

Comment:
Do you know if the compiler or linter will complain about the shorthand hex?

Reply:
The linter has a rule that checks whether hex shorthands are used, and converts it if not.